src/main/java/org/folio/support/ExpirationTool.java

```diff
@@ -69,7 +73,10 @@ private ExpirationTool() {
       .compose(requests -> reorderRequests(conn, vertx, tenant, requests))
       .setHandler(v -> {
         if (v.failed()) {
-          pgClient.rollbackTx(conn, done -> future.fail(v.cause()));
+          pgClient.rollbackTx(conn, done -> {
+            log.error("Error in request processing", v.cause());
```

Comment:
Thanks for this logging! It will help in the future.

Reply:
i hope so, my bad logger wasn't added earlier